src/language/__tests__/schema-kitchen-sink.graphql

```diff
@@ -17,6 +17,7 @@ type Foo implements Bar {
   four(argument: String = "string"): String
   five(argument: [String] = ["string", "string"]): String
   six(argument: InputType = {key: "value"}): Type
+  seven(argument: Int = null): Type
```

Comment:
@cjoudrey

> Since both null and undefined are falsy, I can't think of a case where one would explicitly set the default value to null in order to omit the undefined case.

It is valid language construct. I doesn't care if it is semantically useful - but someone may want explicitly tell that there is no semantical difference between undefined and null